plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TypeConverter.java

```diff
@@ -174,30 +175,52 @@ public static org.apache.iceberg.types.Type toIcebergType(Type type)
         throw new TrinoException(NOT_SUPPORTED, "Type not supported for Iceberg: " + type.getDisplayName());
     }
 
+    private static void validateColumnIdentity(int exceptFields, Optional<ColumnIdentity> columnIdentity)
```

Comment:
@Heltman Could you please explain the meaning of `exceptFields` here?

Reply:
@krvikash Yes, I just want to make sure ColumnIdentity children number is same with child type number in trino type, maybe I don't need t do it.